mavfwd.c

```diff
@@ -143,7 +143,8 @@ static void dump_mavlink_packet(unsigned char *data, const char *direction)
   uint16_t val;
   
 	//RC_CHANNELS ( #65 ) hook
-	if(msg_id == 65 && ch_count > 0) {
+	//RC_CHANNELS_RAW ( #35 ) for ExpressLRS,Crossfire and other RC procotols 
+	if((msg_id == 65 || msg_id == 35) && ch_count > 0) {
```

Comment:
Please consider adding constants first like
#define RC_CHANNELS 65
#define RC_CHANNELS_RAW 35

and replace magic numbers here, after a while nobody will remember what is it 65 and 35 (comments are good, but they tend to rot)